Automattic/newspack-plugin
# feat: custom messaging for reader without password

### All Submissions:






### Changes proposed in this Pull Request:

This PR implements custom messaging for readers that have just created their account and are yet to set their first password:

<img width="504" alt="image" src="https://user-images.githubusercontent.com/820752/188001346-3607734c-0c73-43d9-bd86-8be6f71e1069.png">

The other "reset password" messaging inside the sent email has been updated to use "set a new password" to have a more agnostic feel to it.

<img width="588" alt="image" src="https://user-images.githubusercontent.com/820752/188001394-08a693fc-a268-4374-8d62-fd1af35805c3.png">


### How to test the changes in this Pull Request:

1. Manually remove the saved posts from `newspack_rr_email` post type
2. Create a RAS account and click on the received verification link
3. On the My Account page, confirm the "Create a Password" section as above
4. Click the link to set a new password
5. Set the new password
6. Visit "My Account" and confirm it no longer displays the password creation messaging

### Other information:
